src/patternfly/components/Table/table-td.hbs

```diff
@@ -25,10 +25,21 @@
     data-label="{{table-td--data-label}}"
   {{/if}}>
   {{#unless table-td--IsEmpty}}
-    {{#if table-td--toggle}}
-      <button class="{{pfv}}button pf-m-plain{{#if table-tr--expanded}} pf-m-expanded{{/if}}" {{#if table-td--button--attribute}}{{{table-td--button--attribute}}}{{/if}} {{#if table-tr--expanded}}aria-expanded="true"{{/if}}>
+    {{#if table-td--action}}
+      {{#> button button--IsSmall=table--IsCompact button--IsPlain=true button--id=(concat table--id '-action-kebab-' table-tr--index) button--aria-label="Actions"}}
```

Comment:
It looks like `table-IsCompact` is being used just to trigger a small button. Should it also be used to add the `pf-m-compact` modifier to the table?

Also, I'm wondering if it makes sense to make the button be a menu-toggle instead. Maybe not, since an action could be anything, but here it's used as a toggle for a menu, which makes it seem like it should be a menu-toggle then.

I think a kebab menu is probably the most frequent use case, so it's good to show that but at the same time, it's not the most basic example either. Curious what @mcoker thinks before changing anything. This could also definitely wait for a later day.